data/hai/unfettered side missions.txt

```diff
@@ -160,267 +96,413 @@ mission "Hai Legacy 1"
 			choice
 				`	"I accept."`
-				`	"I have nothing to prove to you."`
-					decline
-			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
-			`	You are not sure if they are being serious or not, but their friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
-			`	"Yeah, what makes you think that, human?" Another one adds.`
+				`	"I have nothing to prove to you. If you want it, you'll have to chase me down. No duel."`
+					goto headingout
+			`	"Good! Of course, I won't be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You're not sure if Mera is being serious or not, but her friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" another one adds.`
 			choice
 				`	(Say nothing.)`
 				`	"Are you trying to scare me out of the fight?"`
 					goto scare
-			`	"I guess we'll find out." Says a third.`
+			`	"I guess we'll find out." says a third.`
 			label scare
-			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. As far as I am concerned, you are an enemy. You should see it more as a fight for you to truly earn this ship, and more, but make no mistakes: it has only two rules: nobody else than us may interfere, and being disabled results in losing the fight. And do not even think about bringing more than this Ladybug as a flagship or we will not bother showing up!"`
-			# this is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
-			# (but in reality with disabling a lot of ships it goes faster than that)
-			`	They suddenly look at you intensively, straight in the eyes, "We sometimes bend the rules for our friends, but that is not your case," they say with a sudden change of tone, before leaving.`
+			`	The leader of the group gets closer and addresses you in a more serious tone. "You should be aware that this is not a friendly competition. This is a fight for you to truly earn your place with this ship. As far as I'm concerned, you are an enemy. Make no mistakes about that. This fight has a few... implicit rules. We would assume everyone know them, but since you are an outsider I will explain them to you: naturally since this is a duel nobody other than us here may interfere, but also know that being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Anomalocaris, or we will not bother showing up! You must prove you deserve this ship, after all.`
```

Comment:
```suggestion
			`	The leader of the group gets closer and addresses you in a more serious tone. "You should be aware that this is not a friendly competition. This is a fight for you to truly earn your place with this ship. As far as I'm concerned, you are an enemy. Make no mistakes about that. This fight has a few... implicit rules. We would assume everyone know them, but since you are an outsider I will explain them to you. Naturally, since this is a duel nobody other than us here may interfere, but also know that being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Anomalocaris, or we will not bother showing up! You must prove you deserve this ship, after all.`
```